Makefile

```diff
@@ -10,5 +10,11 @@ npm_install:
 bundle_install:
 	bundle install
 
+start:
+	bundle exec nanoc view
+
+watch:
+  	bundle exec nanoc live
```

Comment:
```suggestion
	bundle exec nanoc live
```

Reply:
Ugh sorry - I was editing it via the GitHub web UI so didn't spot the spaces...